pages/level-change.js

```diff
@@ -0,0 +1,274 @@
+import { Component } from "react";
+import Link from "next/link";
+import Layout from "../components/layout.js";
+import styled, { css } from "styled-components";
+
+import { getUsers } from "../networkRequests/user";
+import { roleChange } from "../networkRequests/roles";
+class LevelChangePage extends Component {
```

Comment:
So I locally can't get this page to render-so I would suggest doing both of these
1. Add a PR description with how to test and also what you did
2. Add screenshots of the page in the PR request